odoo/addons/base/models/res_partner.py

```diff
@@ -662,6 +662,8 @@ def _get_name(self):
             name = name + "\n" + partner._display_address(without_company=True)
         name = name.replace('\n\n', '\n')
         name = name.replace('\n\n', '\n')
+        if self._context.get('show_db_id'):
```

Comment:
Would use a more specific context key, even if other context keys are not using a prefix. Like "partner_show_db_id".